packages/components/src/LegacyPagination/LegacyPagination.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { MaterialPlayArrow } from '@t3n/icons';
+import { color, typography } from 'styled-system';
+import { Box } from '../Box';
+import { Icon } from '../Icon';
+
+export interface LegacyPaginationProps {
+  page: number;
+  pages: number;
+  onClick: (page: number) => void;
+}
+
+export interface LegacyPaginationContainerProps {
+  disabled?: boolean;
+}
+
+const RotatedIcon = styled(Icon)`
+  transform: rotate(180deg);
+`;
+
+const LegacyPaginationContainer = styled(Box).attrs({
+  display: 'flex',
```

Comment:
Warum machst du das nicht direkt im CSS? :D

Reply:
why not? :D habs aber direkt ins css geschrieben. Also: done